lib/backend.js

```diff
@@ -661,25 +661,59 @@ Backend.prototype.querySubscribe = function(agent, index, query, options, callba
         'DB does not support subscribe'
       ));
     }
-    backend.pubsub.subscribe(request.channel, function(err, stream) {
-      if (err) return callback(err);
+
+    var channels = request.channels;
+
+    if (request.channel) {
+      logger.warn('[DEPRECATED] Do no use channel for query, use channels instead.');
```

Comment:
1. Suggested update to be clearer, since this only shows the log line - `[DEPRECATED] "query" middleware's context.channel is deprecated, use context.channels instead.`
2. Also update query middleware documentation - https://share.github.io/sharedb/middleware/actions#query

Reply:
Changed